src/tests/ftest/launch.py

```diff
@@ -428,6 +433,14 @@ def get_logs_dir(self):
 
         """
         default_base_dir = os.path.join("~", "avocado", "job-results")
+        logger.info("default_base_dir = %s", default_base_dir)
+        datadir_paths_logs_dir = self.get_setting("datadir.paths", "logs_dir", default_base_dir)
+        logger.info("datadir.paths = %s ", datadir_paths_logs_dir)
+        time.sleep(2)
+        datadir_paths_logs_dir = self.get_setting("datadir.paths", "logs_dir", default_base_dir)
+        logger.info("datadir.paths = %s ", datadir_paths_logs_dir)
+        if datadir_paths_logs_dir != '/var/tmp/ftest/avocado/job-results':
+            sys.exit(-1)
```

Comment:
This is debug code, right?  This check probably doesn't make sense outside of our CI environment.